src/graphql/validation/rules/overlapping_fields_can_be_merged.py

```diff
@@ -98,10 +98,7 @@ def enter_selection_set(self, selection_set: SelectionSetNode, *_args: Any) -> N
 # Field name and reason.
 ConflictReason: TypeAlias = Tuple[str, "ConflictReasonMessage"]
 # Reason is a string, or a nested list of conflicts.
-if MYPY:  # recursive types not fully supported yet (/python/mypy/issues/731)
```

Comment:
You can now remove the MYPY constant completely.

Reply:
Removed